### PR TITLE
Dodge with few ms less and velocity grow, same for roll

### PR DESCRIPTION
**Pull request by Meghil**

### CODE REVIEW

File: Assets/Resources/aaaaaatry/Script/RollAndDodgeState.cs [hunk 1]
```diff
@@ -19,11 +19,8 @@ override public void OnStateUpdate(Animator animator, AnimatorStateInfo stateInf
 
     // OnStateExit is called when a transition ends and the state machine finishes evaluating this state
     override public void OnStateExit(Animator animator, AnimatorStateInfo stateInfo, int layerIndex)
-    {
-        if (animator.GetComponent<BaseMove>() != null)
```
Comment (MattiaPezzanoAiv) [inline Assets/Resources/aaaaaatry/Script/RollAndDodgeState.cs]: you may to implement this check, preventing bug.

File: Assets/Resources/aaaaaatry/Script/ToAttackBehaviour.cs [hunk 2]
```diff
@@ -29,7 +29,7 @@ override public void OnStateUpdate(Animator animator, AnimatorStateInfo stateInf
             
             ClientUtilities.SendHeavyAttack(ClientManager.PlayerInstance.transform.position, ClientManager.PlayerInstance.transform.rotation);
         }
-        if (InputBrawler.Instance.GetAction(Action.Action).WasPressed && !animator.GetComponent<BaseMove>().Dodging)
```
Comment (MattiaPezzanoAiv) [inline Assets/Resources/aaaaaatry/Script/ToAttackBehaviour.cs]: reimplement this code string " if (InputBrawler.Instance.GetAction(Action.Action).WasPressed && !animator.GetComponent<BaseMove>().Dodging)", now are fixed.

**Comment by MattiaPezzanoAiv** [comment]
Leaved 2 comments, read and please fix code strings.

File: Assets/Resources/aaaaaatry/Script/RollAndDodgeState.cs (hunk 1 above)
Comment (Meghil) [inline Assets/Resources/aaaaaatry/Script/RollAndDodgeState.cs]: this check is no more necessary because  client roll with ClientRollandDashstate

**Pull request merged**